public/data/yiremorlans.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "Yire Morlans",
+  "type": "personal",
+  "bio": "Full-stack Software Engineer | In my free time, I create apps and play Pokémon Go.",
+  "avatar": "https://github.com/yiremorlans.png",
+  "links": [
+    {
+      "name": "My portfolio website",
+      "url": "https://yiremorlans.netlify.app/",
+      "icon": "link"
+    },
+    {
+      "name": "Follow my Open Source on GitHub",
+      "url": "https://github.com/yiremorlans",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/xenaaMorph",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/yiremorlans/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Join the 100devs community on discord",
```

Comment:
I recommend capitalizing the `d` here. What do you think?
```suggestion
      "name": "Join the 100devs community on Discord",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant

Reply:
Excellent! Thanks for catching that!